### ISSUE TITLE
Nicehash v2.0.1.3 out of memory Issue still 

### ISSUE THREAD

**Issue by gasitup** [nicehash/NiceHashMiner2#173]
Last night the nicehash legacy v1.8.1.3 just stopped working and would not restart even after computer reboot.

I installed Nicehash Beta v2.0.1.3 and it STILL has the out of memory issue after running for about 8 hrs and shuts down the mining.

Running : Windows 10 / ASUS Prime Z270 / Intel Celeron CPU G3930 2.9 GHz x2 / HyperX FURY Black 8 GB 2133 MHz CL14 DIMM DDR4 / 6x Gigabyte G1 8GB 1070's 

Page file is set to MAX

SO my question would be : How much memory do you need ?? or Will this EVER be fixed ??

**Comment by mvisconti** [comment]
It seems like a lot of people are experiencing this issue. I am as well. 

What is your VRAM set too?

**Comment by gasitup** [comment]
not sure what you mean by VRAM 
I use MIS Afterbrner 
Core clock 100
Memory clock 630

**Comment by gasitup** [comment]
so I have added 8gb more ram and increased my page file or Vram to max for 16gb of ram and have not had anymore issues

**Comment by its-arun** [comment]
I have 32 GB ram, 60 GB vram still using more than 6 neoscrypt crashes the system.
Stock clock.

**Comment by gasitup** [comment]
"Hello!

Please note that we will fix this issue with the next release. Till then we suggest you disable NeoScrypt in your miner. Thank you for understanding.

Kind regards,
NiceHash Team"

**Comment by Tottom** [comment]
Hi, 

So i tried 16GB vMem and got the errors more frequently. And on 64GB vMem and had the errors maybe about 20 hours later. 

I have now upped my vMem to 98GB and i have not had any further errors in the last 24 hours. I have 6x 1070 and 2x 1070ti cards. OC 150mhz on CPU and 500mhz on memory and 65% TDP. 

I have only 8GB physical memory.

**Comment by gasitup** [comment]
did you disable NeoScrypt ?? that seems to have been the issue all along

**Comment by Tottom** [comment]
No, NeoScrypt still enabled.

**Comment by Tottom** [comment]
![altcoin](https://user-images.githubusercontent.com/26551123/34463055-f8eaea78-ee59-11e7-941c-2821d0e744a4.PNG)

**Comment by Tottom** [comment]
![altcoin1](https://user-images.githubusercontent.com/26551123/34463064-3ca21796-ee5a-11e7-9d89-285b42203061.PNG)

**Comment by gasitup** [comment]
according to a support reply on another post they suggest it 

Please note that we will fix this issue with the next release. Till then we suggest you disable NeoScrypt in your miner. Thank you for understanding.

Kind regards,
NiceHash Team"

**Comment by gasitup** [comment]
personally I switched to only running Equihash, running all the different ones it kept dropping 1 of my GPS's

**Comment by Tottom** [comment]
I was thinking of doing that too but if Equihash becomes unprofitable then i will need to change it again? Also, i had BSOD once or twice every 24 hours and as soon as NeoScrypt started, it then gave me the memory error, this also resulted in the app looping or the app getting stuck and not closing and restarting as it should. I look forward to the update but the vMem does not impact me, just not sure what it impacts on performance.

**Comment by gasitup** [comment]
I ran Equihash only for about 6 weeks and it did not really impact my weekly earnings it stayed about the same, that is till the hack not switching algos you get steadier performance instead of the up and down and loss of time from switching. At least that was my experience but you have to do what you feel comfortable with. I have not been back on Nicehash yet mining Electroneum now

**Comment by M1r00** [comment]
This is an answerI posted on another post also :
 Hi

I have been struggling with the same issue for more than three weeks. My rig has 06 gtx 1070 and whenever Itried to mine on neoscrypt i have got errors such out of memory , CUDA error on majority of my cards and sometimes random reboot or freezes.
I am running on windows10 x64 1709. and nvidia driver 384.76 (this one seems to be the most stable for me) and my bios is up to date .

I decided to track the issue cause i was suspecting a riser default. so I launched each GPU one by one and tested with equihash, Lyrarev2 and then neoscrypt to detect any drop of performances or power.

When testing my GPU3 which is mounted on riser connected on pcix8 n°2 with neoscrtypt ....I just noticed that the Vram usage was huge, around 5.7GB ....per card!!!!! note that the five other cards were on idle and not mining but their Vram just jumped to 5.7GB

Tested the other cards on neoscrypt the Vram didn't exceed 96MB per card . So I tested nicehash 2 with 5 cards on neoscrypt and GPU 3 on equihash and Lyrarev2 (disabling neoscrypt). the Vram usage was 1.139GB and the rig was more stable. However after two hours found the rig freezed.

I was really disappointed. I tried then Nicehash Legacy with disabling neoscrypt from GPU3 ....EUREKA the rig has been mining for 3 hours now and seems stable ....with very lownusage in cpu 40% (i5 6400).

Hope this will solve your problem with trying to detect which GPU is over using Vram...don'tknow if this is driver, hardware(GPU, motherboard or riser ) or Nicehash/neoscrypt related issue ; I think that  5.7GB vram per card  is unmanageable even with more virtual memory settings.

Regards

**Comment by Tottom** [comment]
Makes sense, thanks for the update. The release of the recent drivers and the NH 2.0.1.5 does not match up maybe? Anyone have an idea if and when there will be an update?

**Comment by M1r00** [comment]
Hi, there.

I think I found a solution this problem, at least for my rig....hope this gonna solve your problem also
here is my config: (the one causing problems)
MSI Z270 A PRO bios 1.51 (latest update)
I5 6400 
12GB DDR4
6x GTX 1070 (1 Aorus+ 5Strix OC) on latest nvidia driver at present time 390.xx
Windows 10 x 64 pro build 1709
SSSd 120GB
2x V750 cooler master PSU.

the other one is the same only but the cpu G3950(7th Gen) this one was rock stable on 24/7 mining

So I decided to use the same settings for soft (BIOS /windows/ driver) that are onthe stable onthe faulty one 

My first step was to export the bios settings from the healthy to the sick rig
several changes were made (see attachement) 
![0191a21a-ae50-43fb-89fc-46e54ecd367a](https://user-images.githubusercontent.com/35006972/34920392-51946438-f972-11e7-938b-b645e8a4aef2.png)

I think (and Ican be wrong ) the crucial parameter is the FCLK as it was originally set to 800Mhz on the faulty rig ...which if we consider the southbridge frequency can be determinent for such heavy usage on the motherboard.
Anyway these changes made my rig rock stable and it is running now for 48h nonstop mining on neoscrypt

My only question  was why the two rigs were not set up in the sam eway as they have the exact same bios and I don't recall when I set up the FCLK??? may be the CPU generation had arole on that ?
Regards.

**Comment by matiasslpknt** [comment]
hi....i had same problem...have 4 aorus 1070 and 1 strix oc...in one moment my rig started mining equihash and neoscrypt...i realized that the miner dont work if i have all in neoscrypts...so i unchecked from mbenchmark of one of the graphics and it starts normally
